backend/src/main/java/com/benine/backend/PresetController.java

```diff
@@ -13,6 +14,8 @@
 
   private ArrayList<Preset> presets = new ArrayList<Preset>();
 
+  private HashSet<String> tags = new HashSet<>();
```

Comment:
I'll remove them from the preset class, I think it's more logical to have them in the preset controller since they're only used for controlling the presets. Do you agree with that?